web/massastation/src/components/SidePanels/Upload/Upload.tsx

```diff
@@ -185,6 +185,10 @@ export default function Upload() {
   const { data: accounts = [] } = useResource<AccountObject[]>(
     `${URL.WALLET_BASE_API}/${URL.WALLET_ACCOUNTS}`,
   );
+  const nicknameInAccounts = accounts.find(
```

Comment:
```suggestion
  const nicknameExistInAccountsList = accounts.find(
```

More explicit 😉

Reply:
IMO for me nicknameExistInAccountsList means this variable is a boolean, true if the nickname exists in account list.